boskos/metrics/metrics.go

```diff
@@ -35,37 +36,44 @@ type prometheusMetrics struct {
 	BoskosState map[string]map[string]prometheus.Gauge
 }
 
+const otherState = "other"
```

Comment:
what's this?

Reply:
So all the metrics that are not defined in states, will be agglomerated as other. This is important for mason since we set leased resources state to the name of the mason resource. ie for resource gke-cluster-01, the state of resource gcp-project-01 will be gke-cluster-01

Reply:
hummm, make it `common.Other`?

Reply:
sure